vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -130,14 +130,17 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.19.9-hotfix.20210526/windowszip/v1.19.9-hotfix.20210526-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.19.11/windowszip/v1.19.11-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.19.11-hotfix.20210526/windowszip/v1.19.11-hotfix.20210526-1int.zip",
+        "https://acs-mirror.azureedge.net/kubernetes/v1.19.12/windowszip/v1.19.12-1int.zip",
```

Comment:
Do we plan to support these new versions or use hotfixes instead? have you synced with @xuto2 and @palma21 ?

Reply:
@mainred  Reference the PR's description
```
Cache this in next AKS Windows VHD so we do not need to release a new AKS Windows VHD when AKS supports above versions. We can replace/remove these binaries when AKS supports above versions with hotfix or skip them.
```

Reply:
but my question is if these versions will be supported? if not, we may need to remove them. You know, we cache **limited** supported versions.

Reply:
We will remove them if they are skipped. This is an exception since we have some ICMs which are impacted by the fixed issue.
```
Cache this in next AKS Windows VHD so we do not need to release a new AKS Windows VHD when AKS supports above versions.
```

Reply:
If they are for icms, I guess hotfixes for other existing k8s versions need also to be included? because usually, we support more than three versions.

Reply:
It is not so urgent for us to prepare the hotfix for the old versions. Since these new k8s patch versions include the fix and normally we should support them soon, I cached these k8s versions for nice to have. I want to avoid that we release the next AKS Windows VHD without these k8s versions and then we need to release a new one. k8s package's size is small so it is not a big concern. If it is an image, we should be more serious.